h5py/h5p.pyx

```diff
@@ -1197,16 +1192,20 @@ cdef class PropFAID(PropInstanceID):
 
     IF MPI:
         @with_phil
-        def set_fapl_mpio(self, Comm comm not None, Info info not None):
+        def set_fapl_mpio(self, comm, info):
```

Comment:
IMHO, this should use a default `info=None` which should be translated to `MPI_INFO_NULL`. Anyway, maybe that should be discussed/done in another PR.

Reply:
That makes sense, but it's not high priority - users wouldn't usually use this API directly, and the higher-level API already sets a default:

https://github.com/h5py/h5py/blob/2660313fcfdb927303d96848014a639b6230abb0/h5py/_hl/files.py#L43-L49